hyperspy/_components/gaussian2d.py

```diff
@@ -48,18 +50,21 @@ def __init__(self,
                  sigma_y=1.,
                  centre_x=0.,
                  centre_y=0.,
+                 rotation=0.,
```

Comment:
`__init__` needs a docstring. The dosctring should explain that `rotation` is given in mrad.